### PR TITLE
Prettier and Auto Deploy

### PR DESCRIPTION
**Pull request by kevin-carlson**
Prettier added and pre-commit githook configured.

Configured autodeploy to skynet on pull request.

### CODE REVIEW

**Review by MSevey — CHANGES_REQUESTED** [review]
As discussed
- [ ] Remove husky and idea folders
- [ ] clean up git history to remove skynet context from this PR

**Review by MSevey — APPROVED** [review]

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
```
Comment (kwypchlo) [inline .github/workflows/deploy.yml:14]:
```suggestion
          node-version: 16.x
```

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  pull_request:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Use Node.js
+        uses: actions/setup-node@v1
```
Comment (kwypchlo) [inline .github/workflows/deploy.yml:12]:
```suggestion
        uses: actions/setup-node@v2
```

File: .github/workflows/deploy.yml [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+          node-version: 14.x
+
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build
+        run: npm run build
+
+      - name: Deploy to Skynet
+        uses: SkynetLabs/deploy-to-skynet-action@v1
```
Comment (kwypchlo) [inline .github/workflows/deploy.yml:23]: I'm not sure @v1 will work, if not then use @main

**Review by kwypchlo — CHANGES_REQUESTED** [review]
looks good, minor improvements :)

**Review by MSevey — APPROVED** [review]

File: .github/workflows/deploy.yml (hunk 3 above)
Comment (kevin-carlson) [reply .github/workflows/deploy.yml:23]: Looks like v1 works

**Review by MSevey — APPROVED** [review]

**Review by kwypchlo — APPROVED** [review]

**Pull request merged**